packages/scoped-custom-element-registry/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@webcomponents/scoped-custom-element-registry",
-  "version": "0.0.3",
+  "version": "0.0.4",
```

```suggestion
  "version": "0.0.3",
```

We bump versions when we release